e2e/sendSheetFlow.spec.js

```diff
@@ -0,0 +1,255 @@
+    await Helpers.checkIfVisible('send-savings-cSAI');
+    await Helpers.tap('send-savings-cSAI');
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('selected-asset-field-input');
+  });
+
+  it('Should go back to Asset List after tapping on savings asset', async () => {
+    await Helpers.delay(1000);
+    await Helpers.tap('send-asset-form-cSAI');
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('send-asset-list');
+  });
+
+  it('Should display Asset Form after tapping on asset', async () => {
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('send-asset-SOCKS');
+    await Helpers.tap('send-asset-SOCKS');
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('selected-asset-field-input');
+  });
+
+  it('Should display max button on asset input focus', async () => {
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('selected-asset-field-input');
+    await Helpers.tap('selected-asset-field-input');
+    await Helpers.delay(1000);
+    await Helpers.checkIfElementByTextIsVisible('Max');
+  });
+
+  it('Should display max button on asset quantity input focus', async () => {
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('selected-asset-quantity-field-input');
+    await Helpers.tap('selected-asset-quantity-field-input');
+    await Helpers.delay(1000);
+    await Helpers.checkIfElementByTextIsVisible('Max');
+  });
+
+  it('Should display Insufficient Funds button if exceeds asset balance', async () => {
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('selected-asset-field-input');
+    await Helpers.tap('selected-asset-field-input');
+    await Helpers.typeText('selected-asset-field-input', '9999', true);
+    await Helpers.delay(1000);
+    await Helpers.checkIfElementByTextIsVisible('Insufficient Funds');
+  });
+
+  it('Should prepend a 0 to quantity field on input of .', async () => {
+    await Helpers.tap('send-asset-form-SOCKS');
+    await Helpers.delay(2000);
+    await Helpers.tap('send-asset-SOCKS');
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('selected-asset-quantity-field-input');
+    await Helpers.tap('selected-asset-quantity-field-input');
+    await Helpers.typeText('selected-asset-quantity-field-input', '.', true);
+    await Helpers.checkIfElementByTextIsVisible('0.');
+  });
+
+  it('Should only show a max of 2 decimals in quantity field', async () => {
+    await Helpers.tap('send-asset-form-SOCKS');
+    await Helpers.delay(1500);
+    await Helpers.tap('send-asset-SOCKS');
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('selected-asset-quantity-field-input');
+    await Helpers.tap('selected-asset-quantity-field-input');
+    await Helpers.typeText(
+      'selected-asset-quantity-field-input',
+      '8.1219',
+      true
+    );
+    await Helpers.checkIfElementByTextIsVisible('8.12');
+    await Helpers.tap('send-asset-form-SOCKS');
+  });
+
```

Comment:
Do you mind duplicating the whole SOCKS logic and do the same with ETH?
Literally going back one step, selecting ETH and duplicating everything you did in the form.

There are a few differences in the way we handle both so it would be very useful to make sure both are tested.